db/seeds.rb

```diff
@@ -1,16 +1,25 @@
-# Создадим записи в базе данных для проверки,
-# восклицательный знак после create стоит для вызова исключения.
-
-Card.create!([
-  { original_text: "Привет. Как дела?",
-    translated_text: "Hello. How are you?",
-    review_date: Date.current },
-
-  { original_text: "Откуда ты?",
-    translated_text: "where are you from?",
-    review_date: Date.yesterday },
-
-  { original_text: "Привет. Как дела?",
-    translated_text: "Hello. How are you?",
-    review_date: Date.new(2015, 02, 12) }
-            ])
+# encoding: utf-8
+require "open-uri"
+require "nokogiri"
+
+
+Card.delete_all
+
+URL = "http://masterrussian.com/vocabulary/most_common_words.htm"
+ORIGINAL_COL = 2
+TRANSLATED_COL = 3
+
+document = Nokogiri::HTML(open(URL))
+rows = document.css("table.topwords").css('tr')[1..-1]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.